src/understanding-deis/concepts.md

```diff
@@ -11,27 +11,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
```

Comment:
We are moving to rename the PaaS layer as "Workflow" and Deis becomes the company name. This opens up space to have additional product lines under the Deis umbrella as with router, database, slugbuilder/runner and future projects like service brokers, identity mgmt, etc. Make sense?